lib/gyp/platform/win.js

```diff
@@ -409,67 +392,164 @@ win.getOSBits = function getOSBits() {
     return 32;
 };
 
-win.genEnvironment = function genEnvironment(outDir, version, arch) {
+function tryVS7_powershell() {
```

Comment:
Camel case? Considering that all of these functions are prefixed with `tryVS7`, should we put them in `win/vs7.js` folder?